cartridges/int_klaviyo_core/cartridge/scripts/klaviyo/eventData/startedCheckout.js

```diff
@@ -35,9 +46,12 @@ function getData(currentBasket) {
             data.line_items.push(productObj);
             data.Categories.push.apply(data.Categories, data.line_items[itemIndex].Categories);
             data.Items.push(data.line_items[itemIndex]['Product Name']);
+
+            reconstructCartItems.push({ productID: currentProductID, quantity, options });
```

Comment:
Should this object be {productID: currentProductID, quantity: quantity, options: options}? I just had some issues with an object in compatibility mode 21.7 where I had module.exports = {function1, function2}; and had to update it to module.exports = {function1: function1, function2: function2};

Reply:
Thanks for this and for showing me how the issue occurred during our call. Just for tracking, reference - confirming this change was made as part of the [follow-up PR](https://github.com/maze-consulting/SFCC_Klaviyo_Maze/pull/17/files). 

That line now reads as: 

`reconstructCartItems.push({ productID: currentProductID, quantity: quantity, options: options });`